src/blocks/Listing/schema.js

```diff
@@ -282,6 +284,28 @@ export const setCardStylingSchema = ({ schema, intl }) => {
       actions: Object.keys(ALIGN_INFO_MAP),
       actionsInfoMap: ALIGN_INFO_MAP,
     },
+    objectfit: {
```

Comment:
@nileshgulia1 You should use either **camelCase** or **snake_case** for 2-words properties names.

Reply:
@avoinea As those fields are used as css classes yield by `styleWrapper`, I don't think we have a notion to use cameCase CSS in design system. Maybe we can use ['object-fit'] ?